src/components/GrowlNotification/types.ts

```diff
@@ -0,0 +1,9 @@
+type GrowlRef = {
```

Comment:
Use type from [Growl.ts](https://github.com/software-mansion-labs/expensify-app-fork/blob/2b955ccd6600591dda3c3b8d0a92fd04ddc554f3/src/libs/Growl.ts) instead

Reply:
I moved it here due to this comment:

https://github.com/Expensify/App/issues/25127#issuecomment-1715359171

Reply:
We agreed with Błażej to keep it where it was without moving it here.